source/manage/cloud-ip-filtering.rst

```diff
@@ -0,0 +1,93 @@
+Cloud IP Filtering
+========================
+
+.. include:: ../_static/badges/ent-cloud-only.rst
+   :start-after: :nosearch:
+
+IP filtering is a powerful security feature that allows system admins to control access to their workspace by defining approved IP ranges. Only users within these specified IP ranges can access the workspace, ensuring enhanced security for your workspace. IP filtering requires a subscription to Mattermost Cloud Enterprise.
+
+Configure IP filtering
```

Comment:
@nickmisasi - Based on the [v9.4 server changelog](https://github.com/mattermost/docs/pull/6825), Cloud IP Filtering will be disabled OOTB behind a feature flag. 

Are you open to updating this new docs page to include details on enabling the feature flag as an admin?

Reply:
End users can't control feature flags in Cloud. The feature flag's in place so that we can wait until other system dependencies (CWS, Provisioner, etc) are ready without having to hold up the 9.4 release. An additional benefit is that we can disable the new feature in the event a glaring issue is found after release. 

I'm not sure it's worth documenting this as it's invisible to the user and may just cause confusion. Let me know what you think!

Reply:
One thing to add - the plan would be to enable the feature flag immediately as soon as the release rolls out to the enterprise ring in Cloud

Reply:
You're absolutely right -- users can't enable feature flags in Cloud. I'd forgotten that important point. No sense documenting something that Cloud admins won't be able to do.

Given that the plan is to enable soon after release to the Enterprise ring, I wonder if the changelog needs a small update to clarify that point? Thoughts?